### PR TITLE
feat(integrationCenter): add cluster type for data source

### PR DESCRIPTION
**Pull request by huxiangtao**
**What this PR does / why we need it**:

外部系统来源 新增集群类型
image

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @caosm @qiuxiaolian @dd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by supereagle** [comment]
@caosm @qiuxiaolian @cd1989   PTAL

File: web/src/components/integration/component/formGroup/Cluster/IsWorkerCluster.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { Field } from 'formik';
+import MakeField from '@/components/public/makeField';
+import PropTypes from 'prop-types';
+import { Form, Input } from 'antd';
+import SwitchField from './SwitchField';
+const InputField = MakeField(Input);
+const FormItem = Form.Item;
+
+export default class IsWorkerCluster extends React.Component {
```
Comment (cd1989) [inline web/src/components/integration/component/formGroup/Cluster/IsWorkerCluster.js]: WorkerCluster ?

**Pull request merged**